src/pages/index.js

```diff
@@ -101,10 +101,6 @@ profileImageButton.addEventListener("click", () => {
   profileImageModal.open();
 });
 
-profileImageSaveButton.addEventListener("click", () => {
-  handleProfileImageForm();
-});
-
 function processLikeClick(card) {
   const isLiked = card.isLiked();
   console.log(isLiked);
```

Comment:
`  this._likeButton.classList.toggle("card__button_active");
`


please, delete it


`this` should **not** be used outside classes in functions in `index.js` . This context is easy to lose. You need to find other ways of passing data.


--------------------


Now only class `Card` knows every card selector, so you need to encapsulate the likes logics inside its methods and call them in `index.js`